File: Diplom/Diplom/Models/CommentModels/MusicCommentResult.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace Diplom.Models.CommentModels
{

    public enum CommentChangedType
    {
        Added = 1,
        Deleted = 2
    }

    public class MusicCommentResult
    {
        public bool Result { get; set; }
        public CommentChangedType CommentChangedType { get; set; }
        public MusicCommentInfo MusicCommentInfo { get; set; } = new MusicCommentInfo();
    }
}

File: Diplom/Diplom/ClientApp/src/app/models/comments/musicCommentResult.ts
import {MusicCommentInfo} from "./musicCommentInfo";

export enum CommentChangedType{
  added = 1,
  deleted = 2,
  edited = 3
}

export class MusicCommentResult {
  result: boolean;
  commentChangedType: CommentChangedType;
  musicCommentInfo: MusicCommentInfo;
}
